service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -178,6 +183,7 @@ public void sendCourtBundlesUploadedNotification(final FurtherEvidenceUploadedEv
         }
     }
 
+    @Async
```

Comment:
Just wondering - do we want these to be @Retryable too?

Reply:
The service will not throw exception if it fails to send notification. It will only log an error message.
So, if the handler catches any exception, it should be some data issue / program bugs (e.g. null pointer) which I think is a bit pointless to retry. 
But sure I can add @Retryable if we want to play safe.

Reply:
yeah makes sense - ok as it is i think